src/pages/App.js

```diff
@@ -63,7 +64,32 @@ export default function App() {
       />
       <button type="submit" onClick={handleSubmit}>Search</button>
       <p>
-        {foods.map((elem) => <li>{elem.description}</li>)}
+        {foods.slice(10, 30).map((elem) => {
+          if (elem.description.toLowerCase() === food.toLowerCase()) {
+            return null;// no need to show it in the page
```

Comment:
Why leave out exact matches? Nothing wrong with it, just curious about this choice.

Reply:
Because the API shows same result that user type for example if user type "beef" the result will show first 5 items just name as "beef".

Reply:
Couldn't you just apply the same set logic to those results? So there's only one result that says exactly "beef".